File: src/main/java/uk/co/alt236/apkcompare/output/PrintabilityEvaluator.java
package uk.co.alt236.apkcompare.output;

import uk.co.alt236.apkcompare.comparators.results.ComparisonResult;
import uk.co.alt236.apkcompare.comparators.results.Similarity;

public class PrintabilityEvaluator {

    public boolean isPrintable(final ComparisonResult item,
                               final boolean verbose) {
        if (verbose) {
            return true;
        }

        return item.getSimilarity() != Similarity.IDENTICAL;
    }
}

File: app/src/main/java/uk/co/alt236/apkcompare/app/output/PrintabilityEvaluator.kt
package uk.co.alt236.apkcompare.app.output

import uk.co.alt236.apkcompare.app.comparators.results.ComparisonResult
import uk.co.alt236.apkcompare.app.comparators.results.Similarity

class PrintabilityEvaluator {

    fun isPrintable(item: ComparisonResult,
                    verbose: Boolean): Boolean {
        return if (verbose) {
            true
        } else item.similarity !== Similarity.IDENTICAL

    }
}
